README.md

```diff
@@ -152,6 +153,35 @@ In order to start using the player you need to add a [YouTubePlayerView](#youtub
 </LinearLayout>
 ```
 
+## Use with Jetpack Compose
+``` kotlin 
+  @Composable
+    fun Player() {
+        val activityLifecycle = lifecycle
+        val context = LocalContext.current
+
+        val youtubePlayer = remember {
+            YouTubePlayerView(context).apply {
+                activityLifecycle.addObserver(this)
+                enableAutomaticInitialization = false
+                initialize(object : AbstractYouTubePlayerListener() {
+                    override fun onReady(@NonNull youTubePlayer: YouTubePlayer) {
+                        youTubePlayer.cueVideo("fV7TZJCa9c8", 0f)
+                    }
+                })
+            }
+        }
+
+        AndroidView(
+            {
+                youtubePlayer
+            }, modifier = Modifier
+                .fillMaxWidth()
+                .wrapContentHeight()
+        )
```

Comment:
Would it make sense to wrap the AndroidView inside a DisposableEffect? Something like this:
```
DisposableEffect(
    AndroidView(
        modifier = Modifier.fillMaxWidth(), 
        factory = { youtubePlayerView }
    ), 
    effect = { onDispose { youtubePlayerView.release() } }
)
```